Blish HUD/GameServices/Modules/Managers/Gw2ApiManager.cs

```diff
@@ -1,39 +1,53 @@
 ﻿using Gw2Sharp.WebApi;
 using System.Collections.Generic;
 using System.Linq;
+using Blish_HUD.Gw2WebApi;
+using Gw2Sharp;
 using Gw2Sharp.WebApi.V2;
 using Gw2Sharp.WebApi.V2.Models;
 
 namespace Blish_HUD.Modules.Managers {
     public class Gw2ApiManager {
 
-        private readonly HashSet<TokenPermission> _permissions;
+        private const int SUBTOKEN_LIFETIME = 7;
 
-        private Connection _gw2ApiConnection;
+        private static readonly List<Gw2ApiManager> _apiManagers = new List<Gw2ApiManager>();
```

Comment:
I would like to propose that we use another naming convention for static fields, so it is clear in the code.

Reply:
Currently we're using _camelCase for all non-public fields.  Naming convention here is indicating public vs. not public, which is fairly common from what I can tell.

We can certainly discuss potential alternatives, but I don't think this PR should be the place for that discussion or ground zero for those changes.

Reply:
agreed